Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -320,7 +320,9 @@ private extension HTTPClient {
                     httpCode: urlResponse?.httpStatusCode ?? response.statusCode
                 ))
             case let .failure(error):
-                Logger.debug(Strings.network.api_request_failed(request.httpRequest, error: error))
+                Logger.debug(Strings.network.api_request_failed(request.httpRequest,
+                                                                httpCode: urlResponse?.httpStatusCode,
```

Comment:
In the success case we fallback to the cached status code. Is there a reason we need that? Just wondering if it's the same here.

Reply:
Because `urlResponse` is optional here, for cases when the response fails catastrophically and there’s nothing. In the case of an error we don’t have a parsed response anyway, only this optional.